public/sass/base/_utilities.scss

```diff
@@ -0,0 +1,301 @@
+					float: none;
+					display: block;
+					text-align: left;
+				}
+			}
+		}
+		@include main-nav-li;
+	}
+	.header__list__right {
+		@include nav-li;
+		@include main-nav-li;
+	}
+	.header__cart {
+		background-color: $bg-color;
+		float: right;
+		position: absolute;
+		left: 80%;
+		top: 0%;
+		&__box {
+			a {
+				position: relative;
+			}
+			p {
+				position: absolute;
+				left: 65%;
+				top: 10%;
+				&:last-child {
+					display: none;
+				}
+			}
+			@include  flex-align-items;
+			justify-content: center;
+			a.icon {
+				display: block;
+				margin-top: 20px;
+			}
+		}
+	}
+	
+	.topnav.responsive {
+		.icon {
+			position: absolute;
+			right: 0;
+			top: 0;
+		}
+	}
+	.mobile-nav {
+		background: $bg-color;
+		li {
+			padding: 5px 0px;
+			width: $width;
+			text-align: center;
+			&:hover {
+				background: $color-secondary;
+				border-bottom: $border-bottom;
+			}
+			a {
+				font-weight: 300;
+				display: block;
+				padding: 12px;
+				color: $color-primary;
+				text-decoration: none;
+				box-shadow: 0 -1px rgba(0, 0, 0, 0.5) inset;
+				transition: background .3s;
+				white-space: nowrap;
+			}
+		}
+	}
+	.header__list {
+		background: $bg-color;
+		display: none;
+	}
+	.signin, .login {
+		margin: 20px;
+		flex-direction: column;
+		align-items: inherit;
+	}
+	.product__container {
+		max-width: 480px;
+		flex-direction: column;
+	}
+	.product__container__item {
+		flex-wrap: nowrap;
+		flex-direction: column;
+	}
+	.product__container__list {
+		height: auto;
+		ul#menu {
+			label {
+				position: relative;
+				display: block;
+				pointer-events: auto;
+				padding: 0 18px 0 12px;
+				line-height: 3em;
+				transition: background 0.3s;
+				cursor: $cursor;
+			}
+			input {
+				display: none;
+			}
+		}
+		ul {
+			li {
+				align-self: center;
+				padding: 0%;
+				border: none;
+			}
+		}
+	}
+	.product__container__list__item {
+		p {
+			font-size: 14px;
+			min-height: 180px;
+			line-height: 24px;
+		}
+	}
+	.item__price {
+		text-align: center;
+		span {
+			display: none;
+		}
+		a {
+			width: $width;
+		}
+		.btn {
+			width: 50%;
+			text-decoration: none;
+			span {
+				position: initial;
+				color: $color-secondary;
+				font-size: 16px;
+				transform: none;
+				&::before {
+					content: " @ ";
+				}
+			}
+		}
+	}
+	ul#menu {
+		background:  $color-primary;
+		color: $color-secondary;
+	}
+	.total__price {
+		align-self: center;
+		flex: 1 0 20%;
+	}
+	label {
+		&:after {
+			content: "";
+			position: absolute;
+			display: block;
+			top: 50%;
+			right: 30px;
+			width: 0;
+			height: 0;
+			border-top: 4px solid rgba(255,255,255,255);
+			border-bottom: 0 solid rgba(255,255,255,255);
+			border-left: 4px solid transparent;
+			border-right: 4px solid transparent;
+			transition: border-bottom .1s, border-top .1s .1s;
+		}
+		&:hover {
+			~ {
+				label {
+					background: rgba(0,0,0,.3);
+				}
+			}
+		}
+	}
+	#check01 {
+		&:checked {
+			~ {
+				label {
+					&:after {
```

Comment:
Try not to add selectors more than 3 levels, use BEM methodology

Reply:
Done.